pennylane/qinfo/classical_fisher.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Differentiable classical fisher information"""
+# pylint: disable=import-outside-toplevel
+import functools
+import pennylane as qml
+
+from pennylane.transforms import batch_transform
+
+
+def _torch_jac(circ):
+    """Torch jacobian as a callable function"""
+    import torch
+
+    def wrapper(*args, **kwargs):
+        loss = functools.partial(circ, **kwargs)
+        if len(args) > 1:
+            return torch.autograd.functional.jacobian(loss, (args))
+        return torch.autograd.functional.jacobian(loss, *args)
+
+    return wrapper
+
+
+def _tf_jac(circ):
+    """TF jacobian as a callable function"""
+    import tensorflow as tf
+
+    def wrapper(*args, **kwargs):
+        with tf.GradientTape() as tape:
+            loss = circ(*args, **kwargs)
+        return tape.jacobian(loss, (args))
+
+    return wrapper
+
+
+def CFIM(qnode, argnums=0):
```

Comment:
```suggestion
def classical_fisher_info(qnode, argnums=0):
```
I am not sure, but I think It would be more clear. Is the matrix necessary?

Reply:
you could even just have `classical_fisher` (since the `info` is somewhat implied, and I tend to prefer fewer underscores where possible).